EdgeFixer/edgefixer.c

```diff
@@ -190,4 +190,8 @@ int main(int argc, const char **argv)
 
 	return 0;
 }
+#else
```

Comment:
The (commented) main threw me off, so I added an empty main so that it could compile as an executable. Clearly it was not necessary.